app/assets/scripts/views/analytics-admin-area.js

```diff
@@ -27,11 +27,16 @@ var AnalyticsAA = React.createClass({
     const done = ids.filter(v => v.inTheDatabase).length;
     const total = ids.length;
     const completion = ((done / total) * 100);
+    const bucket = 'https://s3.amazonaws.com/openroads-vn-dumps/by-province-name/';
+    const file = data.provinceName.match(/ /g) ? data.provinceName.replace(' ', '+') : data.provinceName;
```

Comment:
Huh, both work for me in my web browser:

![plus-vs-space](https://user-images.githubusercontent.com/4959135/28594145-e38f7af6-715c-11e7-829b-7e33c7c1a1e5.gif)